packages/roosterjs-editor-core/lib/editor/createEditorCore.ts

```diff
@@ -43,6 +43,10 @@ export default function createEditorCore(
     let eventHandlerPlugins = allPlugins.filter(
         plugin => plugin.onPluginEvent || plugin.willHandleEventExclusively
     );
+    if (options.darkModeOptions && options.darkModeOptions.transformOnInitialize === undefined) {
+        // Since you can't provide default values on interfaces, provide the default here.
+        options.darkModeOptions.transformOnInitialize = true;

```

Comment:
I think we'd better not directly change the value from options since it is created by caller, and caller is able to change it after creating editor, which may cause unpredictable issue.

Reply:
We'd better clone a copy of darkModeOptions then we are safe to do any modification

Reply:
This setting technically only matters the first time the editor is initialized... Does that change anything?

Reply:
hmm, functionally yes, it should work. But still feel we should not change the option value directly. In an extreme case, say user save the option object on a mobx store, and pass it into editor, then we changed it, and this will cause a runtime exception.